apps/simulators/src/app/app.component.html

```diff
@@ -8,49 +8,44 @@
   [healthy]="healthy$ | async">
   <!-- Start Hero Banner -->
   <biosimulations-topbar-menu>
-    <biosimulations-hover-open-menu [disabled]="true">
+    <biosimulations-hover-open-menu [disabled]="true" [routerLink]="['/simulators']">
       <biosimulations-topbar-menu-item
         heading="BioSimulators"
         icon="simulator"
         id="trigger"
-        [routerLink]="['/simulators']"
         target="_self">
       </biosimulations-topbar-menu-item>
     </biosimulations-hover-open-menu>
 
-    <biosimulations-hover-open-menu [disabled]="true">
+    <biosimulations-hover-open-menu [disabled]="true" [link]="config.dispatchAppUrl">
       <biosimulations-topbar-menu-item
         heading="Run Simulations"
         icon="experiment"
         id="trigger"
-        [href]="config.dispatchAppUrl"
         target="_self">
       </biosimulations-topbar-menu-item>
     </biosimulations-hover-open-menu>
 
     <biosimulations-hover-open-menu>
       <biosimulations-topbar-menu-item heading="Utilities" icon="construction" id="trigger">
       </biosimulations-topbar-menu-item>
-
       <ng-container id="items">
-        <biosimulations-dropdown-menu-item heading="Convert a file" [routerLink]="['/utils', 'convert-file']">
+        <biosimulations-dropdown-menu-item heading="Convert a file" [href]="config.utilsConvertFileUrl">
```

Comment:
I don't understand why you would replace a relative link (which always stays within the app) with a configured absolute link.

Reply:
Ugh. I did not in fact do this intentionally: somehow, just this was copied over in a change from fbd39dc8a37b42d445a9bb03ca0ff9005c27e541, where you adjusted the routing for items within `platform`. This is not applicable (of course) to this app. Fixing now!